### PR TITLE
Improved dust limit computation for BTC and BTC Alts

### PR DESCRIPTION
**Pull request by lvndry**
### 📝 Description

- Improved calculation for BTC and altcoins Dust limit

### ❓ Context

- **Impacted projects**: `LLC` 
- **Linked resource(s)**: `-` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledger-live-common/src/families/bitcoin/js-getTransactionStatus.ts [hunk 1]
```diff
@@ -13,8 +13,9 @@ import type { Account } from "@ledgerhq/types-live";
 import type { Transaction, TransactionStatus } from "./types";
 import { calculateFees, validateRecipient, isTaprootRecipient } from "./cache";
 import { TaprootNotActivated } from "./errors";
-import cryptoFactory from "./wallet-btc/crypto/factory";
+import * as utils from "./wallet-btc/utils";
```
Comment (henrily-ledger) [inline libs/ledger-live-common/src/families/bitcoin/js-getTransactionStatus.ts:16]: import  { computeDustAmount } is not better if you only have this ?

**Review by hzheng-ledger — APPROVED** [review]
looks good to me

**Review by haammar-ledger — APPROVED** [review]
Thanks @hzheng-ledger for the suggestion :+1:

File: libs/ledger-live-common/src/families/bitcoin/js-getTransactionStatus.ts (hunk 1 above)
Comment (haammar-ledger) [reply libs/ledger-live-common/src/families/bitcoin/js-getTransactionStatus.ts:16]: @lvndry ^ just in case

**Comment by gre** [comment]
Screenshot 2022-10-31 at 16 08 11

this PR can't be merged at the moment because do not check the requirements.

also, you may want to run the bot against this PR to confirm all works as expected?

**Comment by lvndry** [comment]
Followed the documentation [here](https://github.com/LedgerHQ/ledger-live/wiki/LLC:bot) to run the bot locally but it failed with error `got Invariant Violation: COINAPPS is not set`

@gre

**Comment by lvndry** [comment]
Bot run: https://github.com/LedgerHQ/ledger-live/actions/runs/3411140678

**Pull request merged**